src/currency-field/CurrencyField.ts

```diff
@@ -0,0 +1,377 @@
+import { html, css } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import { live } from 'lit/directives/live.js';
+import _ from 'lodash';
```

Comment:
Consider importing just `{ debounce }` from lodash instead of using default import?

Reply:
Updating as requested